modal/functions.py

```diff
@@ -794,6 +795,7 @@ async def volume_loader():
                 stub_name=stub_name,
                 is_builder_function=is_builder_function,
                 allow_concurrent_inputs=allow_concurrent_inputs,
+                worker_id=_worker_id,
```

Comment:
wdyt about just reading this from an environment variable here? like `config.get(...)`

Reply:
yeah I was thinking that. Only reason against it was that it could be nice in theory to pin multiple functions to different workers for debugging (e.g. CPU and GPU). But maybe that's too much of a stretch.